### PR TITLE
Fix test for preset flags for chrome

### PR DESCRIPTION
**Pull request by ankur22**

### CODE REVIEW

File: chromium/browser_type_test.go [hunk 1]
```diff
@@ -111,17 +109,6 @@ func TestBrowserTypePrepareFlags(t *testing.T) {
 			changeK6Opts:  &k6lib.Options{},
 			expChangedVal: nil,
 		},
-		{
-			flag:       "enable-use-zoom-for-dsf",
```
Comment (ankur22) [inline chromium/browser_type_test.go:115]: Oops, we all missed this 😅

Comment (ka3de) [reply chromium/browser_type_test.go:115]: Oh tricky case, I was wondering why it was not failing in CI or in my local machine, then a saw the skip when `runtime.GOOS != "darwin"`.

**Review by ka3de — APPROVED** [review]
LGTM.

File: chromium/browser_type_test.go (hunk 1 above)
Comment (ankur22) [reply chromium/browser_type_test.go:115]: That'll teach me for not running tests 😅

**Review by inancgumus — APPROVED** [review]

**Comment by inancgumus** [comment]
@ankur22, since you're not around, I'm merging this as it's a constant pain for me while testing locally.

**Pull request merged**